modules/network_load_balancer/main.tf

```diff
@@ -0,0 +1,79 @@
+resource "aws_lb" "tfe_lb" {
+  name               = "${var.friendly_name_prefix}-tfe-web-alb"
+  internal           = var.load_balancing_scheme
+  load_balancer_type = "network"
+  subnets            = var.network_public_subnets
```

Comment:
I added this to our takeaways for follow-up work since the same logic needs to be addressed in the original load balancer submodule: only the public load balancer should be deployed in public subnets.